playwright/plugins/homeserver/index.ts

```diff
@@ -53,7 +53,7 @@ export interface StartHomeserverOpts {
 
 export interface Homeserver {
     start(opts: StartHomeserverOpts): Promise<HomeserverInstance>;
-    stop(): Promise<void>;
+    stop(): Promise<string[]>;
```

Comment:
documentation please!

```suggestion
    /**
     * Stop this test homeserver instance.
     *
     * @returns A list of fully-qualified path names for logfiles generated during this test run.
     */
    stop(): Promise<string[]>;
```

Reply:
```suggestion
    /**
     * Stop this test homeserver instance.
     *
     * @returns A list of relative path names for logfiles generated during this test run.
     */
    stop(): Promise<string[]>;
```